### PR TITLE
Fix: Scroll to picker regression

### PR DESCRIPTION
**Pull request by chrispader**
### Details

Fixes a regression where refs `ScrollViewWithContext` are not forwarded to `ScrollView` component.

Additionally this introduces a mechanism to conditionally set if the `ScrollContext` should be used. This PR adds two props (`scrollToOverflowEnabled` and `scrollContextEnabled`) to the `<Form />` component, so that the `ScrollContext` only gets used if there's some sort of nested `<Picker />` in the form. (Can also be `StatePicker` or `LocalePicker`)

### Fixed Issues

$ https://github.com/Expensify/App/issues/13909
PROPOSAL: https://github.com/Expensify/App/issues/13909#issuecomment-1396792646

### Tests

1. Log in any account
2. Navigate to Settings > Any workspace > Connect bank account > Connect Manually > Press Save&Continue without any data
3. Will not show “fix the errors” press on ‘fix the error’ in the console anymore

---

1. Go to a the workspace settings
2. Go to the "Reimburse expenses" setting
3. Under "Track Distance", tap on the input for "Unit"
5. For iOS, the entire unit field should be visible above the picker. For all other platforms, there should be no changes to the input or the picker (they will all have different pickers).

---

1. Login in New Dot
2. Go to Settings > Payments
3. Click "Add payment method" and select "Debit card"
4. Click "Save" without entering any data
5. Verify that all inputs have errors
6. if you are in web or desktop, resize the window enough to have scrolling in the RHN:

206823060-51332d2e-cde0-4c9f-b7d1-24e329c075c1

7. Scroll down and click "fix the errors"
8. Verify that the first input gets focused and you are scrolled until it is fully visible (including the top border)
9. Input something in the first input "Name on card"
10. Scroll down and click again "fix the errors"
11. Verify that the second input gets focused and you are scrolled until it is fully visible (including the top border)
12. Go back to the Settings > Payments page
13. Click "Add payment method" and select "Bank account"
14. Verify that the spinner is in the middle and not in the top

206823150-e009919f-f7e7-4a58-9d69-22bf55211798

---

1. Login in New Dot
2. Go to Settings -> Workspaces -> Any workspace -> Connect bank account -> Connect manually
3. Leave inputs empty
4. Click on "Continue"
5. Page shouldn't bounce up to the top, like in this video:

https://user-images.githubusercontent.com/20173411/213497053-73d24ca0-38dd-4df9-8144-76f5872766e0.mov

---

- [x] Verify that no errors appear in the JS console

### Offline tests

Same as regular tests.

### QA Steps

Same as regular tests.

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

No UI changes.

</details>

<details>
<summary>Mobile Web - Chrome</summary>

No UI changes.

</details>

<details>
<summary>Mobile Web - Safari</summary>

No UI changes.

</details>

<details>
<summary>Desktop</summary>

No UI changes.

</details>

<details>
<summary>iOS</summary>

No UI changes.

</details>

<details>
<summary>Android</summary>

No UI changes.

</details>

### CODE REVIEW

**Comment by aldo-expensify** [comment]
I'm not sure if this is caused by this PR yet but there seem to be some dark overlay that doesn't allow me to click on a unit in Android:

image

I was following [this QA](https://github.com/Expensify/App/pull/13514) in Android

**Comment by aldo-expensify** [comment]
I think it would be good to also include in the test/qa steps the steps from these other PR to make sure this is still working: 
- https://github.com/Expensify/App/pull/13514
- https://github.com/Expensify/App/pull/13499

And the steps from this comment to make sure it doesn't bounce weirdly in small forms: https://github.com/Expensify/App/issues/13909#issuecomment-1379483718

**Comment by chrispader** [comment]
> I think it would be good to also include in the test/qa steps the steps from these other PR to make sure this is still working:
> 
> * [Fix: Scroll to Picker TextInput on open #13514](https://github.com/Expensify/App/pull/13514)
> * [Use measureLayout to calculate scrolling #13499](https://github.com/Expensify/App/pull/13499)
> 
> And the steps from this comment to make sure it doesn't bounce weirdly in small forms: [#13909 (comment)](https://github.com/Expensify/App/issues/13909#issuecomment-1379483718)

done!

**Comment by aldo-expensify** [comment]
I tested this in web/ios/android, following the QA (check that the error is not there anymore), and also checking this: https://github.com/Expensify/App/pull/14409#issuecomment-1397193509 and I didn't spot any problem. Things seems to be fixed.

I still [requested](https://expensify.slack.com/archives/C03TQ48KC/p1674144938663599) another pair of eyes to review the PR because I'm not very familiarized with this.

**Comment by chrispader** [comment]
> I'm not sure if this is caused by this PR yet but there seem to be some dark overlay ~that doesn't allow me to click on a unit~ (correction: I can click the unit despite the dark overlay) in Android:
> 
> image
> 
> I was following [this QA](https://github.com/Expensify/App/pull/13514) in Android
> 
> Update: This seems to be like this in `main` too

You're right, i'm getting this on `main` as well. Seems to be some other bug, but definitely not related to this PR

File: src/pages/settings/Payments/AddDebitCardPage.js
Comment (luacmartins) [inline src/pages/settings/Payments/AddDebitCardPage.js:124]: I think that we have to add these props to a few more pages, e.g. [CompanyStep](https://github.com/Expensify/App/blob/7ba22bdc256ad16f517102d8a12db7137c7a150a/src/pages/ReimbursementAccount/CompanyStep.js), [AdditionalDetailsStep](https://github.com/Expensify/App/blob/7ba22bdc256ad16f517102d8a12db7137c7a150a/src/pages/EnablePayments/AdditionalDetailsStep.js), otherwise we have the issue with the Picker being hidden in iOS:

https://user-images.githubusercontent.com/22219519/213524511-8a01f6ed-3d9f-4cbc-b9b2-bcba35f784f6.mov

**Review by luacmartins — CHANGES_REQUESTED** [review]
Great start! I left a few comments. Additionally, there are two bugs that are also on main, but maybe we should address them in a different PR.

**No error displayed for input in short screens**

1. In either iOS or Android, navigate to `Settings > Workspace > Connect bank account > Connect manually`
2. Have the keyboard active
3. Press `Continue`
4. Notice that the error does not show for `Account number` and the `Terms checkbox`
5. You can reproduce on web too, if you resize the browser to a short height.

https://user-images.githubusercontent.com/22219519/213525060-448afa38-f1e9-492d-bf92-23422595dc41.mov

https://user-images.githubusercontent.com/22219519/213525074-1b677044-3cd3-47ce-a212-69be8ad6acf7.mov

**Form scrolls to random position on submit with errors**
1. Navigate to `Settings > Workspace > Connect bank account > Connect manually > Company step`
2. Press `Save & continue` without entering values
3. Verify that the errors show but the form is scrolled to a random position.

**Comment by thesahindia** [comment]
Works well! Some changes are needed as mentioned by @luacmartins

File: src/components/Form.js
Comment (chrispader) [reply src/components/Form.js:59]: sure!

File: src/components/Form.js
Comment (chrispader) [reply src/components/Form.js:69]: feel free to re-write the docs on this. Maybe i overcomplicated the text a bit 😅

**Comment by chrispader** [comment]
> Great start! I left a few comments. Additionally, there are two bugs that are also on main, but maybe we should address them in a different PR.
> 
> **No error displayed for input in short screens**
> 
> 1. In either iOS or Android, navigate to `Settings > Workspace > Connect bank account > Connect manually`
> 2. Have the keyboard active
> 3. Press `Continue`
> 4. Notice that the error does not show for `Account number` and the `Terms checkbox`
> 5. You can reproduce on web too, if you resize the browser to a short height.

Maybe this got fixed already on main, because i can't reproduce this on my emulator after merging main into this branch (at approx. 01/20/2023 9am CET):

**Android (emulator):**

[android recording.webm](https://user-images.githubusercontent.com/20173411/213664446-045d990d-5b96-4ecf-a468-43c103085530.webm)

**Web:**

https://user-images.githubusercontent.com/20173411/213665109-f22acdd4-fa8f-426e-bc10-f4301f482597.mov

File: src/pages/settings/Payments/AddDebitCardPage.js
Comment (chrispader) [reply src/pages/settings/Payments/AddDebitCardPage.js:124]: Done! I added it to all remaining pages/forms that contains (a) picker(s). In [27f5fea](https://github.com/Expensify/App/pull/14409/commits/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140), i also added it to some pages/forms, that seem like they are currently not in use, such as [BankAccountPlaidStep.js](https://github.com/margelo/expensify-app-fork/blob/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140/src/pages/ReimbursementAccount/BankAccountPlaidStep.js)

File: contributingGuides/FORMS.md
Comment (luacmartins) [inline contributingGuides/FORMS.md:279]:
```suggestion

### Handling nested Pickers in Form

In case there's a nested Picker in Form, we should pass the props below to Form, as needed:

#### Enable ScrollContext

Pass the `scrollContextEnabled` prop to enable scrolling up when Picker is pressed, making sure the Picker is always in view and doesn't get covered by virtual keyboards for example.

#### Enable scrolling to overflow

In addition to the `scrollContextEnabled` prop, we can also pass `scrollToOverflowEnabled` when the nested Picker is at the bottom of the Form to prevent the popup selector from covering Picker.
```

**Review by luacmartins — COMMENTED** [review]
Left a couple of suggestions for the docs. 

Additionally, I think we might need to pass the props to the following pages too:

- [AddPersonalBankAccountPage](https://github.com/Expensify/App/blob/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140/src/pages/AddPersonalBankAccountPage.js#L99-L106) since AddPlaidBankAccount has a nested Picker
- [RequestorStep](https://github.com/Expensify/App/blob/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140/src/pages/ReimbursementAccount/RequestorStep.js#L159-L165) since IdentityForm has a nested Picker
- [WorkspaceSettingsPage](https://github.com/Expensify/App/blob/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140/src/pages/workspace/WorkspaceSettingsPage.js#L80-L87)

**Comment by chrispader** [comment]
@luacmartins do you know what the `FormScrollView` component is used for essentially?

For example, we also have in `ACHContractStep`, we also have a `IdentityForm` wrapped by a `ReimburseAccountForm` component. (which renders the `FormScrollView`)

I'm thinking if it's necessary to also add the two additional props from `Form` to [FormScrollView](https://github.com/Expensify/App/blob/fd5e781707ce8860fae9fe09a96654ad0fe49297/src/components/FormScrollView.js#L11) to also enable this functionality here

**Comment by luacmartins** [comment]
@chrispader as far as I know `FormScrollView` should be deprecated in favor of Form - we are refactoring `ACHContractStep` to use Form [here](https://github.com/Expensify/App/pull/13501) and we shouldn't introduce any new instances of it. I don't think that we need to worry about the other existing pages using `FormScrollView` because they don't seem to have nested Pickers.

**Review by luacmartins — DISMISSED** [review]
LGTM!

**Comment by aldo-expensify** [comment]
Thanks @luacmartins for the thorough review! I'll do my review later today.

File: src/components/Form.js
Comment (thesahindia) [inline src/components/Form.js:67]: NAB: Just need to add some spaces
```suggestion
    /** Whether the ScrollView overflow content is scrollable.
     *   Set to true to avoid nested Picker components at the bottom of the Form from rendering the popup selector over Picker
     *   e.g. https://github.com/Expensify/App/issues/13909#issuecomment-1396859008
     */
    scrollToOverflowEnabled: PropTypes.bool,

    /** Whether ScrollWithContext should be used instead of regular ScrollView.
     *  Set to true when there's a nested Picker component in Form.
     */
```

**Comment by thesahindia** [comment]
@chrispader, I am seeing this error -

https://user-images.githubusercontent.com/83179295/214135220-e56675b8-1617-4fc8-9cca-1f00b16884ce.mov

**Comment by aldo-expensify** [comment]
Sorry guys, I failed to get to review this today. I'll get to it first thing tomorrow

**Comment by chrispader** [comment]
> 

Fixed! I just forgot to change the ref variable in two calls in `Form`

**Review by luacmartins — APPROVED** [review]

**Comment by thesahindia** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/83179295/214418838-79974924-bb6b-4aa8-9756-114a25c1bfbb.mov




</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/83179295/214418890-c19af3dd-2242-4ece-98a4-77d1bd7d487b.mov




</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/83179295/214418904-38ae291e-e42a-41c9-b340-faef5a47c3bd.mov





</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/83179295/214418913-b87fa3df-4b37-4ff9-9636-4c722cab67d2.mov




</details>

<details>
<summary>iOS</summary>



https://user-images.githubusercontent.com/83179295/214418927-1dabb5f4-765a-4420-8e89-549e1643bed2.mov



</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/83179295/214419786-702fbf33-ce09-4c04-87ab-43091850efb7.mov




</details>

**Review by thesahindia — APPROVED** [review]
Looks good to me!

**Review by aldo-expensify — APPROVED** [review]
Tested well for me! thanks!

File: src/components/Form.js
Comment (aldo-expensify) [reply src/components/Form.js:308]: I agree, that would have resulted in nicer code 🥲

**Pull request merged**
